### PR TITLE
doc: Updates to Go builder README

### PR DESCRIPTION
**Pull request by laurentsimon**

### CODE REVIEW

File: internal/builders/go/README.md [hunk 1]
```diff
@@ -29,13 +30,13 @@ The following [GitHub trigger events](https://docs.github.com/en/actions/using-w
 - `release`
 - Manual run via `workflow_dispatch`
 
-However, in practice, most triggers should work with the exception of `pull_request`. If you would like support for `pull_request`, please tell us about your use case on [issue #358](https://github.com/slsa-framework/slsa-github-generator/issues/358). If you have an issue in all other triggers please submit a [new issue](https://github.com/slsa-framework/slsa-github-generator/issues/new/choose).
+In practice, most triggers should work with the exception of `pull_request`. If you would like support for `pull_request`, please tell us about your use case on [issue #358](https://github.com/slsa-framework/slsa-github-generator/issues/358). If you have an issue in all other triggers please submit a [new issue](https://github.com/slsa-framework/slsa-github-generator/issues/new/choose).
```
Comment (olivekl) [inline internal/builders/go/README.md:33]: Nit: "If you have an issue **with any** other triggers..."

File: internal/builders/go/README.md [hunk 2]
```diff
@@ -91,15 +92,13 @@ builds:
...
+  # This instructs GoReleaser to not build for linux amd64.
   ignore:
     - goos: linux
       goarch: amd64
 ```
 
-We think gradual adoption is good for projects to get used to SLSA.
-
-The configuration file accepts many of the common fields Goreleaser uses, as you can see in the [example](#configuration-file). The configuration file also supports two variables: `{{ .Os }}` and `{{ .Arch }}`. Other variables can be set manually as shows in the table below, in combination with the builder's `evaluated-envs`:
+The configuration file accepts many of the common fields GoReleaser uses, as you can see in the [example](#configuration-file). The configuration file also supports two variables: `{{ .Os }}` and `{{ .Arch }}`. Other variables can be set manually as shows in the table below, in combination with the builder's `evaluated-envs`:
```
Comment (olivekl) [inline internal/builders/go/README.md:101]: "as **shown** in the table below"

Comment (laurentsimon) [reply internal/builders/go/README.md:101]: good catch, done.

File: README.md [hunk 3]
```diff
@@ -24,7 +24,8 @@ This repository contains the code, examples and technical design for system desc
 
 ## Generation of provenance
 
-Below we describe the various builders and generators in this repository.
+Below we describe the various builders and generators in this repository. They let you build and / or generate non-forgeable provenance
+using a trusted / isolated re-usable workflow. You can read up on the design in our [technical design document](#technical-design).
 
 **Note**: At present the GitHub Actions provided in this repository as builders and generators **MUST** be referenced by tag in order for the `slsa-verifier` to be able to verify the ref of the trusted builder/generator's reusable workflow.
```
Comment (olivekl) [inline README.md:30]: Consider breaking this into two sentences for readability.

**Review by olivekl — APPROVED** [review]
Left a few small comments, lgtm.

File: README.md (hunk 3 above)
Comment (laurentsimon) [reply README.md]: updated, PTAL

Comment (olivekl) [reply README.md]: Great, lgtm!

**Pull request merged**